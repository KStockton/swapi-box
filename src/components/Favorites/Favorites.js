import React from 'react'
import './Favorites.scss'

const Favorites = () => {

  return (
    <div className="Favorites-btn">
      <p>View Favorites</p>
      <p>0</p>
    </div>
  )
}

export default Favorites