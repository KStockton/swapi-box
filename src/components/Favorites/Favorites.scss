.Favorites-btn {
  display: flex;
    flex-direction: row;
    align-items: center;
    border: 2px solid white;
    width: 12rem;
    padding-left: 15px;
    border-radius: 12px;
    margin: 26px;
    font-size: 19px;
    text-align: center
}
.Favorites-btn:hover{
  color: yellow;
}

.Favorites-click {
 margin-left: 3px
}