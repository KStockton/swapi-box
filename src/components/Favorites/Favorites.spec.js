import React from 'react';
import Favorites from './Favorites';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({ adapter: new Adapter() });

describe('Favorites', () =>{
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Favorites />)
  })
  it('Expect component to render without crashing', () =>{
    expect(wrapper).toMatchSnapshot()
  })
})