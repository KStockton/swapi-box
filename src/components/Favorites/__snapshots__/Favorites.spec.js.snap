// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites Expect component to render without crashing 1`] = `
<div
  className="Favorites-btn"
>
  <p>
    View Favorites
  </p>
  <p>
    0
  </p>
</div>
`;
